src/rpc/misc.cpp

```diff
@@ -347,11 +347,11 @@ UniValue verifymessage(const JSONRPCRequest& request)
             "\nUnlock the wallet for 30 seconds\n"
             + HelpExampleCli("walletpassphrase", "\"mypassphrase\" 30") +
             "\nCreate the signature\n"
-            + HelpExampleCli("signmessage", "\"1D1ZrZNe3JUo7ZycKEYQQiQAWd9y54F4XZ\" \"my message\"") +
+            + HelpExampleCli("signmessage", "\"1D1ZrZNe3JUo7ZycKEYQQiQAWd9y54F4XX\" \"my message\"") +
```

Comment:
Interesting. Though, I wanted to make the minimal change possible (I changed one letter) to keep things simpler. Do you think calling another function is necessary for this? It would be confusing if the address changes on each run for example.